common.go

```diff
@@ -99,43 +96,11 @@ func (api *Client) Init(cfg *Config) error {
 		cfg.ApiVersion = 1
 	}
 	api.Config = cfg
-	api.headers = make(map[string]string)
-
-	if api.Client == nil {
```

Comment:
Removing this block - callers can supply their own `http.Client` with the same sort of workaround if necessary.